RNTester/js/ProgressBarAndroidExample.android.js

```diff
@@ -16,33 +16,31 @@ var createReactClass = require('create-react-class');
 var RNTesterBlock = require('RNTesterBlock');
 var RNTesterPage = require('RNTesterPage');
 
-var MovingBar = createReactClass({
-  displayName: 'MovingBar',
-  _intervalID: (null: ?IntervalID),
+class MovingBar extends React.Component<{}> {
+  displayName = 'MovingBar';
+  _intervalID = IntervalID ? IntervalID : null;
```

Comment:
I don't think this is valid. I think it should be

```
_intervalID = (null: ?IntervalID)
```

Reply:
@gugakatsi `IntervalID` is a type (and not a JavaScript object). Since JavaScript doesn't understand Flow types, we can't assign Flow types to JavaScript properties, which is why this is invalid. Also, what we really want is what @TheSavior suggested.